web_ui/frontend/app/api/docs/pelican-swagger.yaml

```diff
@@ -485,17 +495,79 @@ paths:
               initialized:
                 type: boolean
                 example: true
+  /auth/cilogon/login:
+    get:
+      tags:
+        - auth
+      summary: Redirect user to CILogon authentication page for OAuth2 third-party login
+      parameters:
+        - in: query
+          name: next_url
+          type: string
+          description: The path to redirect user to once they successfully authenticated against CILogon
+      responses:
+        "307":
+          description: Redirect user to CILogon authentication page
+        "500":
+          description: Internal server error when failed to generate CSRF cookie for the OAuth flow
```

Comment:
I'd change to "Internal server error when generating CSRF cookie for OAuth flow"